src/bun.js/test/jest.zig

```diff
@@ -1789,6 +1789,37 @@ fn formatLabel(globalThis: *JSC.JSGlobalObject, label: string, function_args: []
                     // ignore unrecognized fmt
                 },
             }
+        } else if (char == '$' and (idx + 1 < label.len) and !(args_idx >= function_args.len)) {
+            var arg = function_args[args_idx];
+
+            while (arg.isObject()) {
+                const remaining = label[idx + 1 ..];
+                const dot_idx = strings.indexOfChar(remaining, '.') orelse remaining.len;
+                const space_idx = strings.indexOfChar(remaining, ' ') orelse remaining.len;
+
+                if (dot_idx < space_idx) {
+                    const property = remaining[0..dot_idx];
+                    if (property.len == 0) break;
+
+                    idx += property.len;
+                    if (arg.get(globalThis, property)) |value| {
```

Comment:
i hate to be a downer but this will need an exception check, something we don't do nearly enough but is very plausible in this case

`if (globalThis.hasException())` and then if yes return .zero probably